### ISSUE TITLE
Any Plans for updates on Slap-Dash

### ISSUE THREAD

**Issue by nima-yz** [VirginiaTech/Slapdash-Server#2]
@amhokies Do you have any plans to work on SlapDash? If so I will be able to work with you as well. I am more down for working in two weeks period of development and bug fixes

**Comment by amhokies** [comment]
Yeah I'm looking into migrating from Mongo to SQL using Sequelize, then I have plans to setup continuous integration with Travis.CI, and try to get Docker working properly.

**Comment by nima-yz** [comment]
Maybe we should have a skype meeting to see for possible opportunities, What do you think?

**Comment by amhokies** [comment]
Are you still in Blacksburg? Maybe we can just meet in the CS lounge or
something.

**Comment by nima-yz** [comment]
I am actually back in Nova area. I work full-time now but still available anytime after 6pm any day.
